lxd/container_lxc.go

```diff
@@ -1264,6 +1273,44 @@ func (c *containerLXC) startCommon() (string, error) {
 					return "", err
 				}
 			}
+		} else if m["type"] == "nic" {
+			if m["nictype"] == "bridged" && shared.IsTrue(m["security.mac_filtering"]) {
```

Comment:
Maybe, though so far we've usually been ignoring options that don't make sense rather than failing at validation.

The reason is that otherwise using "set/unset" gets painful as you wouldn't be able to switch nictype without also unsetting and setting all the right things at once, which you can't unless you're using edit.